crates/starknet-os/src/hints/os.rs

```diff
@@ -84,3 +84,135 @@ pub fn set_ap_to_new_block_hash(
 
     Ok(())
 }
+
+#[cfg(test)]
+mod tests {
+    use std::collections::HashMap;
+
+    use cairo_vm::hint_processor::hint_processor_definition::HintReference;
+    use cairo_vm::serde::deserialize_program::ApTracking;
+    use cairo_vm::types::errors::math_errors::MathError;
+    use cairo_vm::types::exec_scope::ExecutionScopes;
+    use cairo_vm::types::relocatable::Relocatable;
+    use cairo_vm::vm::vm_core::VirtualMachine;
+    use cairo_vm::Felt252;
+    use rstest::{fixture, rstest};
+
+    use super::*;
+    use crate::utils::get_variable_from_root_exec_scope;
+
+    #[fixture]
+    fn os_input() -> StarknetOsInput {
+        StarknetOsInput {
```

Comment:
I think that StarknetOsInput doesn't derive from Default but that might be handy for other tests. Besides, it will look cleaner if the variable is initialized only with the needed values

Reply:
Note that the fixture here is different than what `#[derive(Default)]` would be (`new_block_hash`, `prev_block_hash`, `full_output` are not defaults)

Reply:
I don't understand your request. The struct needs to be initialized with all fields and has no new method to return an empty struct that I can fill.